crates/oxide/src/extractor/named_variant_machine.rs

```diff
@@ -235,6 +235,22 @@ impl Machine for NamedVariantMachine<ParsingState> {
                 //             ^
                 Class::Colon => return self.done(self.start_pos, cursor),
 
+                // A dot must be surrounded by numbers
+                //
+                // E.g.: `px-2.5`
```

Comment:
Should we use a variant example here instead of a utility value example?

Reply:
Copy paste before going to the gym oops. Will fix, thanks!